test/trento/application/integration/discovery/policies/sap_system_policy_test.exs

```diff
@@ -86,6 +89,58 @@ defmodule Trento.Integration.Discovery.SapSystemPolicyTest do
             ]} =
              "sap_system_discovery_application_diagnostics"
              |> load_discovery_event_fixture()
-             |> SapSystemPolicy.handle()
+             |> SapSystemPolicy.handle([])
+  end
+
+  describe "delta deregistration" do
+    test "should deregister the old instances and register the new ones" do
```

Comment:
Could we have some application instances here?
To see if the application deregister command is sent.
At the end, one possible scenario is to have app and database instances in the same host (this is a real case)

Reply:
Sure!